internal/cli/cli.go

```diff
@@ -103,25 +103,17 @@ func parseCmd() *cobra.Command {
 		},
 	}
 
-	var documentGetId string
-	var documentGetShareId string
+	var isShareID bool
 	documentGetCmd := &cobra.Command{
 		Use:   "get",
-		Short: "Get a document",
-		Long:  "Get information about the document. You can either put the documentId or the sharedId as flag",
-		Args:  cobra.MinimumNArgs(0),
+		Short: "Get an existing document by its ID",
+		Long:  "Get information about an existing document by specifying its document ID or a share ID",
+		Args:  cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if documentGetId != "" {
-				return documentGetById(cfg.serverUrl, cfg.apiKey, outline.DocumentID(documentGetId))
-			}
-			if documentGetShareId != "" {
-				return documentGetByShareId(cfg.serverUrl, cfg.apiKey, outline.DocumentShareID(documentGetShareId))
-			}
-			return fmt.Errorf("you should either provide --id or --sharedId")
+			return documentGet(cfg.serverUrl, cfg.apiKey, args[0], isShareID)
 		},
 	}
-	documentGetCmd.Flags().StringVar(&documentGetId, "id", "", "The id of the document")
-	documentGetCmd.Flags().StringVar(&documentGetShareId, "shareId", "", "The sharedId of the document")
+	documentGetCmd.Flags().BoolVar(&isShareID, "share", false, "Treat the argument as document share iD")
```

Comment:
Maybe better going with `share_id` (or `shareId`) as flag name? 🤷 I'm fine with just `share` 😂

Reply:
Thinking from UX perspective, smaller less cryptic options with good documented string are better IMO. I would have gone even further and just called it `s` but stopped in the middle :laughing: 

I always try to follow: 'software is for the user not for developers' so user facing parts should be as simple as possible